<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="memory_size_line">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>128</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>内存大小：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="seg_size_line">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>125</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>段表寄存器大小：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>75</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>段号最大长度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="seginx_line">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>32</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>125</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>段最大长度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="segsize_line">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>32</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="memory_view">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>内存情况：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chg_set_btn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>更改</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>进程名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="job_name">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>215</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>进程1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_job_btn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>放入进程</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="job_view">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>831</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="job_combo">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>作业情况：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>基本设置：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="job_seg_num">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>215</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>进程分段数</string>
    </property>
   </widget>
   <widget class="QComboBox" name="seg_combo">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>270</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>分段：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="req_seg_btn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请求</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>进程名称：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_address_btn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查看地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>段号S</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="s_line">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>340</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>偏移量d</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="d_line">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QComboBox" name="job_combo_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rel_seg_btn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>290</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>释放</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="eliminate">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>请求时淘汰分区</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rel_job_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>释放进程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rel_job_all_btn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>释放所有</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
